warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -289,11 +300,17 @@ public int run(String[] args) throws Exception {
             log.error("No data types were configured");
             return -1;
         }
+        stop = System.currentTimeMillis();
+        log.trace("Time for TypeRegistry.getInstance=" + (stop - start));
         
+        start = System.currentTimeMillis();
         if (!registerTableNames(conf)) {
             return -1;
         }
+        stop = System.currentTimeMillis();
+        log.trace("Time for !registerTableNames=" + (stop - start));
```

Comment:
This is the time to read all the tables from all the ingest components and then register them under job.table.names. I would either drop the not (!) here. The thing you are timing i sjust the registerTableNames() function. The not is used to make sure that it succeeded.